config.py

```diff
@@ -32,10 +33,14 @@
 # SQS QUEUE INFORMATION:
 SQS_QUEUE_NAME = APP_NAME + 'Queue'
 SQS_MESSAGE_VISIBILITY = 1*60           # Timeout (secs) for messages in flight (average time to be processed)
-SQS_DEAD_LETTER_QUEUE = 'arn:aws:sqs:some-region:111111100000:DeadMessages'
+SQS_DEAD_LETTER_QUEUE = 'user_DeadMessages'
```

Comment:
It's possible I missed it, but can you check if the corresponding run.py code to handle this change was added? I didn't see it.

Reply:
already in run.py line 22, from #142